test/sqllogictest/pg_catalog_proc.slt

```diff
@@ -18,31 +18,30 @@ ORDER BY oid
 substring  20003  936  NULL  25
 substring  20003  937  NULL  25
 
-query TII
-SELECT oprname, oid, oprresult
+query TIIII
+SELECT oprname, oid, oprresult, oprleft, oprright
 FROM pg_catalog.pg_operator
 WHERE oprname = '+'
 ORDER BY oid
 ----
-+  550  21
-+  551  23
-+  586  700
-+  591  701
-+  684  20
-+  1076  1114
-+  1327  1184
-+  1337  1186
-+  1360  1114
-+  1363  1114
-+  1758  1700
-+  1800  1083
-+  1849  1186
-+  2066  1114
-+  2551  1114
-+  2553  1186
-+  2554  1186
-+  3898  3831
-+  16433  2276
-+  16466  16460
-+  16467  16462
-+  16468  16464
++  550  21  21  21
++  551  23  23  23
++  586  700  700  700
++  591  701  701  701
++  684  20  20  20
++  1076  1114  1082  1186
++  1327  1184  1184  1186
++  1337  1186  1186  1186
++  1360  1114  1082  1083
++  1363  1114  1083  1082
++  1758  1700  1700  1700
++  1800  1083  1083  1186
++  1849  1186  1186  1083
++  2066  1114  1114  1186
++  2551  1114  1186  1082
++  2553  1186  1186  1114
++  2554  1186  1186  1184
++  3898  3831  3831  3831
++  16466  16460  16460  16460
++  16467  16462  16462  16462
++  16468  16464  16464  16464
```

Comment:
Don't consider unary operators anymore, seems to be consistent with postgres

Reply:
This doesn't seem to be completely true. For example I can create a unary operator and have it show up:
```
postgres=# CREATE FUNCTION complex_add(complex, complex)
    RETURNS complex
    AS 'filename', 'complex_add'
    LANGUAGE C IMMUTABLE STRICT;
ERROR:  type complex does not exist
postgres=# CREATE FUNCTION plus_one(integer) RETURNS integer AS 'select $1 + 1' LANGUAGE SQL IMMUTABLE RETURNS NULL ON NULL INPUT;
CREATE FUNCTION
postgres=# SELECT plus_one(1);
 plus_one 
----------
        2
(1 row)

postgres=# CREATE OPERATOR @ (rightarg = integer, function = plus_one);
CREATE OPERATOR

postgres=# SELECT * FROM pg_operator WHERE oprname = '@';
  oid   | oprname | oprnamespace | oprowner | oprkind | oprcanmerge | oprcanhash | oprleft | oprright | oprresult | oprcom | oprnegate |   oprcode   | oprrest | oprjoin 
--------+---------+--------------+----------+---------+-------------+------------+---------+----------+-----------+--------+-----------+-------------+---------+---------
    473 | @       |           11 |       10 | l       | f           | f          |       0 |       20 |        20 |      0 |         0 | int8abs     | -       | -
    590 | @       |           11 |       10 | l       | f           | f          |       0 |      700 |       700 |      0 |         0 | float4abs   | -       | -
    595 | @       |           11 |       10 | l       | f           | f          |       0 |      701 |       701 |      0 |         0 | float8abs   | -       | -
    682 | @       |           11 |       10 | l       | f           | f          |       0 |       21 |        21 |      0 |         0 | int2abs     | -       | -
    773 | @       |           11 |       10 | l       | f           | f          |       0 |       23 |        23 |      0 |         0 | int4abs     | -       | -
   1763 | @       |           11 |       10 | l       | f           | f          |       0 |     1700 |      1700 |      0 |         0 | numeric_abs | -       | -
 755888 | @       |         2200 |       10 | l       | f           | f          |       0 |       23 |        23 |      0 |         0 | plus_one    | -       | -
(7 rows)

```

My unary operator is at the bottom, but those other `@` symbols are other unary operators. In general it seems that unary operators have an `oprleft` oid of 0.

```
postgres=# SELECT typname FROM pg_type WHERE oid = 0;
 typname 
---------
(0 rows)

postgres=# SELECT * FROM pg_operator WHERE oprleft = 0 OR oprright = 0;
  oid   | oprname | oprnamespace | oprowner | oprkind | oprcanmerge | oprcanhash | oprleft | oprright | oprresult | oprcom | oprnegate |     oprcode     | oprrest | oprjoin 
--------+---------+--------------+----------+---------+-------------+------------+---------+----------+-----------+--------+-----------+-----------------+---------+---------
    473 | @       |           11 |       10 | l       | f           | f          |       0 |       20 |        20 |      0 |         0 | int8abs         | -       | -
    484 | -       |           11 |       10 | l       | f           | f          |       0 |       20 |        20 |      0 |         0 | int8um          | -       | -
    513 | @@      |           11 |       10 | l       | f           | f          |       0 |      603 |       600 |      0 |         0 | box_center      | -       | -
    558 | -       |           11 |       10 | l       | f           | f          |       0 |       23 |        23 |      0 |         0 | int4um          | -       | -
    559 | -       |           11 |       10 | l       | f           | f          |       0 |       21 |        21 |      0 |         0 | int2um          | -       | -
    584 | -       |           11 |       10 | l       | f           | f          |       0 |      700 |       700 |      0 |         0 | float4um        | -       | -
    585 | -       |           11 |       10 | l       | f           | f          |       0 |      701 |       701 |      0 |         0 | float8um        | -       | -
    590 | @       |           11 |       10 | l       | f           | f          |       0 |      700 |       700 |      0 |         0 | float4abs       | -       | -
    595 | @       |           11 |       10 | l       | f           | f          |       0 |      701 |       701 |      0 |         0 | float8abs       | -       | -
    596 | |/      |           11 |       10 | l       | f           | f          |       0 |      701 |       701 |      0 |         0 | dsqrt           | -       | -
    597 | ||/     |           11 |       10 | l       | f           | f          |       0 |      701 |       701 |      0 |         0 | dcbrt           | -       | -
    682 | @       |           11 |       10 | l       | f           | f          |       0 |       21 |        21 |      0 |         0 | int2abs         | -       | -
    773 | @       |           11 |       10 | l       | f           | f          |       0 |       23 |        23 |      0 |         0 | int4abs         | -       | -
    797 | #       |           11 |       10 | l       | f           | f          |       0 |      602 |        23 |      0 |         0 | path_npoints    | -       | -
    799 | @-@     |           11 |       10 | l       | f           | f          |       0 |      602 |       701 |      0 |         0 | path_length     | -       | -
    969 | @@      |           11 |       10 | l       | f           | f          |       0 |      601 |       600 |      0 |         0 | lseg_center     | -       | -
    971 | @@      |           11 |       10 | l       | f           | f          |       0 |      604 |       600 |      0 |         0 | poly_center     | -       | -
   1336 | -       |           11 |       10 | l       | f           | f          |       0 |     1186 |      1186 |      0 |         0 | interval_um     | -       | -
   1420 | @@      |           11 |       10 | l       | f           | f          |       0 |      718 |       600 |      0 |         0 | circle_center   | -       | -
   1521 | #       |           11 |       10 | l       | f           | f          |       0 |      604 |        23 |      0 |         0 | poly_npoints    | -       | -
   1528 | ?-      |           11 |       10 | l       | f           | f          |       0 |      601 |        16 |      0 |         0 | lseg_horizontal | -       | -
   1529 | ?|      |           11 |       10 | l       | f           | f          |       0 |      601 |        16 |      0 |         0 | lseg_vertical   | -       | -
   1591 | @-@     |           11 |       10 | l       | f           | f          |       0 |      601 |       701 |      0 |         0 | lseg_length     | -       | -
   1614 | ?-      |           11 |       10 | l       | f           | f          |       0 |      628 |        16 |      0 |         0 | line_horizontal | -       | -
   1615 | ?|      |           11 |       10 | l       | f           | f          |       0 |      628 |        16 |      0 |         0 | line_vertical   | -       | -
   3147 | ~       |           11 |       10 | l       | f           | f          |       0 |      829 |       829 |      0 |         0 | macaddr_not     | -       | -
   3368 | ~       |           11 |       10 | l       | f           | f          |       0 |      774 |       774 |      0 |         0 | macaddr8_not    | -       | -
   2634 | ~       |           11 |       10 | l       | f           | f          |       0 |      869 |       869 |      0 |         0 | inetnot         | -       | -
   1751 | -       |           11 |       10 | l       | f           | f          |       0 |     1700 |      1700 |      0 |         0 | numeric_uminus  | -       | -
   1763 | @       |           11 |       10 | l       | f           | f          |       0 |     1700 |      1700 |      0 |         0 | numeric_abs     | -       | -
   1794 | ~       |           11 |       10 | l       | f           | f          |       0 |     1560 |      1560 |      0 |         0 | bitnot          | -       | -
   1877 | ~       |           11 |       10 | l       | f           | f          |       0 |       21 |        21 |      0 |         0 | int2not         | -       | -
   1883 | ~       |           11 |       10 | l       | f           | f          |       0 |       23 |        23 |      0 |         0 | int4not         | -       | -
   1889 | ~       |           11 |       10 | l       | f           | f          |       0 |       20 |        20 |      0 |         0 | int8not         | -       | -
   1916 | +       |           11 |       10 | l       | f           | f          |       0 |       20 |        20 |      0 |         0 | int8up          | -       | -
   1917 | +       |           11 |       10 | l       | f           | f          |       0 |       21 |        21 |      0 |         0 | int2up          | -       | -
   1918 | +       |           11 |       10 | l       | f           | f          |       0 |       23 |        23 |      0 |         0 | int4up          | -       | -
   1919 | +       |           11 |       10 | l       | f           | f          |       0 |      700 |       700 |      0 |         0 | float4up        | -       | -
   1920 | +       |           11 |       10 | l       | f           | f          |       0 |      701 |       701 |      0 |         0 | float8up        | -       | -
   1921 | +       |           11 |       10 | l       | f           | f          |       0 |     1700 |      1700 |      0 |         0 | numeric_uplus   | -       | -
   3682 | !!      |           11 |       10 | l       | f           | f          |       0 |     3615 |      3615 |      0 |         0 | tsquery_not     | -       | -
 755888 | @       |         2200 |       10 | l       | f           | f          |       0 |       23 |        23 |      0 |         0 | plus_one        | -       | -
(42 rows)

```

Reply:
Thanks, good catch, I should have checked instead of assuming.